src/Microsoft.AspNetCore.Mvc.TagHelpers/EnvironmentTagHelper.cs

```diff
@@ -11,10 +11,14 @@ namespace Microsoft.AspNetCore.Mvc.TagHelpers
     /// <summary>
     /// <see cref="ITagHelper"/> implementation targeting &lt;environment&gt; elements that conditionally renders
     /// content based on the current value of <see cref="IHostingEnvironment.EnvironmentName"/>.
+    /// If the environment is not listed in explicitly identified <see cref="Names"/> or <see cref="Include"/> lists or
```

Comment:
`If the environment is not listed in <see cref="Names"/> or <see cref="Include"/>, or if it is in <see cref="Exclude"/>, the content will not be rendered.`